basics/getting_started.md

```diff
@@ -3,18 +3,22 @@ layout: contribute
 title: Getting Started
 ---
 
-# Setting up your coding environment
+# Getting started
 
-For now we have support for IntelliJ. We don't have IDE support for other
-languages in Bazel right now.
+This page helps you set up your coding environment, describes creating an
+IntelliJ project and takes you through compiling and debugging your project.
+Finally, you can read about Bazel's source code, how to navigate through it, and
+how to use the Bazel CI system.
 
-## Preparations
+## Installing Bazel
 
 *  [Install Bazel](https://bazel.build/versions/master/docs/install.html) on your
-   system. Note that for developing Bazel, you need the latest released version
-   of Bazel.
+   system. Note that you need the latest released version of Bazel for
+   development.
 *  Clone Bazel's Git repository from GitHub:
+
    *  `git clone https://github.com/bazelbuild/bazel.git`
+
 *  Try to build Bazel (you might need to [install some
```

Comment:
These should be separate items because they are two separate tasks. 

3. Install any missing [prerequisites](https://docs.bazel.build/versions/master/install-compile-source.html#bootstrap-unix).
4. Try to [build Bazel](https://docs.bazel.build/versions/master/guide.html#building-programs-with-bazel):

Reply:
Done.